packages/clay-tabs/docs/tabs.mdx

```diff
@@ -5,12 +5,14 @@ lexiconDefinition: 'https://liferay.design/lexicon/core-components/tabs/'
 packageNpm: '@clayui/tabs'
 ---
 
-import {Tabs, TabsDropdown} from '$packages/clay-tabs/docs/index';
+import {Tabs, ModernTabs, TabsDropdown} from '$packages/clay-tabs/docs/index';
 
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
 -   [Introduction](#introduction)
+-   [Basic Tabs](#basic-tabs)
+-   [Modern Tabs](#modern-tabs)
```

Comment:
Since we no longer want to support the old style or others styles, we can deprecate the `displayType` APIs and remove the old css styles.

Reply:
If we are deprecating the `displayType` api can we also deprecate the `modern` api?

Reply:
@ethib137 yes, we can do that.